[Bug] Why does the latest node Stable release not have npm

```
# dong @ lufei in E:\test [13:04:39]
$ scoop reset nodejs-lts
Resetting nodejs-lts (20.13.0).
Linking D:\scoop_user_apps\apps\nodejs-lts\current => D:\scoop_user_apps\apps\nodejs-lts\20.13.0
Persisting bin
Persisting cache
# dong @ lufei in E:\test [13:09:08]
$ node -v
v20.13.0
# dong @ lufei in E:\test [13:09:12]
$ npm -v
node:internal/modules/cjs/loader:1148
  throw err;
  ^

Error: Cannot find module 'D:\scoop_user_apps\persist\nodejs-lts\bin\node_modules\npm\bin\npm-cli.js'
    at Module._resolveFilename (node:internal/modules/cjs/loader:1145:15)
    at Module._load (node:internal/modules/cjs/loader:986:27)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:174:12)
    at node:internal/main/run_main_module:28:49 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}

Node.js v20.13.0
# dong @ lufei in E:\test [13:09:15] C:1
```

## Bug Report

#### Current Behavior
<!-- A clear and concise description of the behavior. -->

#### Expected Behavior
<!-- A clear and concise description of what you expected to happen. -->

#### Additional context/output
<!-- Add any other context about the problem here. If applicable, paste terminal output here to help explain. -->

#### Possible Solution
<!--- Only if you have suggestions on a fix for the bug -->

### System details

**Windows version:** [e.g. 7, 8, 10, 11]
11
**OS architecture:** [e.g. 32bit, 64bit, arm64]

**PowerShell version:** [output of `"$($PSVersionTable.PSVersion)"`]
PowerShell 7.4.2
**Additional software:** [(optional) e.g. ConEmu, Git]

#### Scoop Configuration
<!-- Can be found in  ~/.config/scoop/config.json -->

```json
//# Your configuration here
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -2,7 +2,7 @@
 
 ### Features
 
-- **scoop-search:** Use SQLite for caching apps to speed up local search ([#5851](https://github.com/ScoopInstaller/Scoop/issues/5851), [#5918](https://github.com/ScoopInstaller/Scoop/issues/5918), [#5946](https://github.com/ScoopInstaller/Scoop/issues/5946), [#5949](https://github.com/ScoopInstaller/Scoop/issues/5949), [#5955](https://github.com/ScoopInstaller/Scoop/issues/5955))
+- **scoop-search:** Use SQLite for caching apps to speed up local search ([#5851](https://github.com/ScoopInstaller/Scoop/issues/5851), [#5918](https://github.com/ScoopInstaller/Scoop/issues/5918), [#5946](https://github.com/ScoopInstaller/Scoop/issues/5946), [#5949](https://github.com/ScoopInstaller/Scoop/issues/5949), [#5955](https://github.com/ScoopInstaller/Scoop/issues/5955), [#5966](https://github.com/ScoopInstaller/Scoop/issues/5966))
 - **core:** New cache filename format ([#5929](https://github.com/ScoopInstaller/Scoop/issues/5929))
 
 ### Bug Fixes

--- a/lib/database.ps1
+++ b/lib/database.ps1
@@ -40,31 +40,6 @@ function Get-SQLite {
 
 <#
 .SYNOPSIS
-    Close a SQLite database.
-.DESCRIPTION
-    Close a SQLite database connection.
-.PARAMETER InputObject
-    System.Data.SQLite.SQLiteConnection
-    The SQLite database connection to close.
-.INPUTS
-    System.Data.SQLite.SQLiteConnection
-.OUTPUTS
-    None
-#>
-function Close-ScoopDB {
-    [CmdletBinding()]
-    param (
-        [Parameter(Mandatory, ValueFromPipeline)]
-        [System.Data.SQLite.SQLiteConnection]
-        $InputObject
-    )
-    process {
-        $InputObject.Dispose()
-    }
-}
-
-<#
-.SYNOPSIS
     Create a new SQLite database.
 .DESCRIPTION
     Create a new SQLite database connection and create the necessary tables.
@@ -164,6 +139,8 @@ function Set-ScoopDBItem {
             $dbTrans.Rollback()
             throw $_
         } finally {
+            $dbCommand.Dispose()
+            $dbTrans.Dispose()
             $db.Dispose()
         }
     }
@@ -292,6 +269,7 @@ function Select-ScoopDBItem {
         [void]$dbAdapter.Fill($result)
     }
     end {
+        $dbAdapter.Dispose()
         $db.Dispose()
         return $result
     }
@@ -353,6 +331,7 @@ function Get-ScoopDBItem {
         [void]$dbAdapter.Fill($result)
     }
     end {
+        $dbAdapter.Dispose()
         $db.Dispose()
         return $result
     }

--- a/lib/database.ps1
+++ b/lib/database.ps1
@@ -40,22 +40,16 @@ function Get-SQLite {
 
 <#
 .SYNOPSIS
-    Create a new SQLite database.
+    Open Scoop SQLite database.
 .DESCRIPTION
-    Create a new SQLite database connection and create the necessary tables.
-.PARAMETER PassThru
-    System.Management.Automation.SwitchParameter
-    Return the SQLite database connection.
+    Open Scoop SQLite database connection and create the necessary tables if not exists.
 .INPUTS
     None
 .OUTPUTS
-    None
-    Default
-
     System.Data.SQLite.SQLiteConnection
     The SQLite database connection if **PassThru** is used.
 #>
-function New-ScoopDB ([switch]$PassThru) {
+function Open-ScoopDB {
     # Load System.Data.SQLite
     if (!('System.Data.SQLite.SQLiteConnection' -as [Type])) {
         try {
@@ -87,11 +81,7 @@ function New-ScoopDB ([switch]$PassThru) {
     )"
     $tableCommand.ExecuteNonQuery() | Out-Null
     $tableCommand.Dispose()
-    if ($PassThru) {
-        return $db
-    } else {
-        $db.Dispose()
-    }
+    return $db
 }
 
 <#
@@ -116,7 +106,7 @@ function Set-ScoopDBItem {
     )
 
     begin {
-        $db = New-ScoopDB -PassThru
+        $db = Open-ScoopDB
         $dbTrans = $db.BeginTransaction()
         # TODO Support [hashtable]$InputObject
         $colName = @($InputObject | Get-Member -MemberType NoteProperty | Select-Object -ExpandProperty Name)
@@ -254,7 +244,7 @@ function Select-ScoopDBItem {
     )
 
     begin {
-        $db = New-ScoopDB -PassThru
+        $db = Open-ScoopDB
         $dbAdapter = New-Object -TypeName System.Data.SQLite.SQLiteDataAdapter
         $result = New-Object System.Data.DataTable
         $dbQuery = "SELECT * FROM app WHERE $(($From -join ' LIKE @Pattern OR ') + ' LIKE @Pattern')"
@@ -310,7 +300,7 @@ function Get-ScoopDBItem {
     )
 
     begin {
-        $db = New-ScoopDB -PassThru
+        $db = Open-ScoopDB
         $dbAdapter = New-Object -TypeName System.Data.SQLite.SQLiteDataAdapter
         $result = New-Object System.Data.DataTable
         $dbQuery = 'SELECT * FROM app WHERE name = @Name AND bucket = @Bucket'

